openfisca_country_template/variables/housing.py

```diff
@@ -24,3 +24,19 @@ class rent(Variable):
     entity = Household
     definition_period = MONTH
     label = u"Rent paid by the household"
+
+
+HOUSING_STATUS = Enum([
```

Comment:
I renamed to `HOUSING_OCCUPANCY_STATUS` for consistency, and added a comment to explain what it is.